### ISSUE TITLE
InsecureRequestWarning: Unverified HTTPS request is being made.

### ISSUE THREAD

**Issue by chevonmdphillip** [swisskyrepo/GraphQLmap#4]
```
root@kali-murdockz:~/GraphQLmap# python3 graphqlmap.py -u "https://[target-site]/graphiql/?query={}"
   _____                 _      ____  _
  / ____|               | |    / __ \| |
 | |  __ _ __ __ _ _ __ | |__ | |  | | |     _ __ ___   __ _ _ __
 | | |_ | '__/ _` | '_ \| '_ \| |  | | |    | '_ ` _ \ / _` | '_ \
 | |__| | | | (_| | |_) | | | | |__| | |____| | | | | | (_| | |_) |
  \_____|_|  \__,_| .__/|_| |_|\___\_\______|_| |_| |_|\__,_| .__/
                  | |                                       | |
                  |_|                                       |_|
                              Author: @pentest_swissky Version: 1.0
GraphQLmap > dump
dump
/usr/local/lib/python3.7/dist-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
[!] Expecting value: line 1 column 1 (char 0)
GraphQLmap >
```

**Comment by fuomag9** [comment]
I have the same issue

**Comment by fuomag9** [comment]
Fixed in f0b190a86e9195d1612b79cc441b49733461f014
